### ISSUE TITLE
Buid failing for .NET nanoFramework

### ISSUE THREAD

**Issue by josesimoes** [angularsen/UnitsNet#962]
**Describe the bug**
- Build is failing after #958 was merged.

**Comment by josesimoes** [comment]
Issue identified. The references are invalid.

This is what's there now:

```xml
    <Reference Include="mscorlib, Version=1.10.5.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
      <HintPath>..\packages\nanoFramework.CoreLibrary.1.10.5\lib\mscorlib.dll</HintPath>
      <Private>True</Private>
      <SpecificVersion>True</SpecificVersion>
    </Reference>
```

This is what's should be there:

![image](https://user-images.githubusercontent.com/1881520/129343237-93c5ea4d-2bb9-41ef-be8a-b78efdcb7629.png)

The revision number isn't being properly updated when the references are updated.
This requires a fix in the update references code to properly handle this.

**Comment by josesimoes** [comment]
For a quick fix on the build breaking I'll manually fix the references and will address the update process in another PR.

**Comment by josesimoes** [comment]
@angularsen to update the references in .NET nanoFramework projects we don't need to it programatically anymore.
Support for nfproj has been added to nuget CLI and that's available since [v10](https://docs.microsoft.com/en-gb/nuget/release-notes/nuget-5.10#summary-whats-new-in-510) which ships with Visual Studio 2019 version 16.10.

Current AppVeyor images include VS 2019 16.10.4 which should have it.

If you're OK with it, I'll remove the current code that's handling these ref updates with a simple 
`nuget update UnitsNet.NanoFramework\GeneratedCode\UnitsNet.nanoFramework.sln`

**Comment by angularsen** [comment]
@josesimoes sounds great to me, let's see if that works.
